packages/opentelemetry/src/utils/parseSpanDescription.ts

```diff
@@ -156,16 +156,16 @@ export function descriptionForHttpMethod(
     opParts.push('prefetch');
   }
 
-  const { urlPath, url, query, fragment, hasRoute } = getSanitizedUrl(attributes, kind);
+  const [parsedUrl, httpRoute] = getParsedUrl(attributes);
 
-  if (!urlPath) {
+  if (!parsedUrl) {
     return { ...getUserUpdatedNameAndSource(name, attributes), op: opParts.join('.') };
```

Comment:
h: If we have an httpRoute but no parsedUrl we would return the original name here. I don't know if that is desired.